cmd/kogito/command/test/common.go

```diff
@@ -74,3 +85,24 @@ func ExecuteCli() (string, string, error) {
 
 	return testOut.String(), testErr.String(), err
 }
+
+func InitConfigWithTestConfigFile(){
```

Comment:
Please add docs to the exported function

Reply:
updated